libdevice/cmath_wrapper.cpp

```diff
@@ -69,6 +69,10 @@ float floorf(float x) { return __devicelib_floorf(x); }
 DEVICE_EXTERN_C_INLINE
 float scalbnf(float x, int n) { return __devicelib_scalbnf(x, n); }
 
+extern "C" SYCL_EXTERNAL float __devicelib_scalblnf(float, long int);
```

Comment:
Is this line required? I think it's not, since this function is declared in `device_math.h`, which is included by this file.